### PR TITLE
Bug 1187836 - [RTL][Bluetooth] Use bdi for string that may have its own directionality, r=evelyn

### PR DESCRIPTION
**Pull request by gasolin**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=af0ff431119dd324093f1b481a1e5c9ed3b2586f

File: apps/settings/js/panels/bluetooth/bt_template_factory.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ define(function() {
   function btTemplate(deviceType, onItemClick, observableItem) {
     var device = observableItem;
 
-    var nameSpan = document.createElement('span');
+    var nameSpan = document.createElement('bdi');
```
Comment (gasolin) [inline apps/settings/js/panels/bluetooth/bt_template_factory.js]: bt_template_factory is used for both "Paired devices" and "Devices in the Area" section, so we just need patch once

Comment (gasolin) [inline apps/settings/js/panels/bluetooth/bt_template_factory.js]: will rename to nameElement

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b95278c9448465c84d0988aace6d34199d0cf71e

**Pull request merged**